server/src/main/java/org/elasticsearch/cluster/SnapshotDeletionsInProgress.java

```diff
@@ -140,6 +137,9 @@ public Version getMinimalSupportedVersion() {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startArray(TYPE);
         for (Entry entry : entries) {
+            if (entry.snapshot.getSnapshotId() == null) {
```

Comment:
I didn't see much point rendering the cleanup entry here (and in `toString`) since it's not actually a delete.